Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -1809,11 +1809,13 @@ namespace Remotion.BocAutoCompleteReferenceValue
             maxHeight = '';
         }
 
-        const availableWidth = position.left + LayoutUtility.GetOuterWidth(reference);
+        const bufferFromLeft = 30;
+        const availableWidth = position.left + LayoutUtility.GetOuterWidth(reference) - bufferFromLeft;
         const minWidth = LayoutUtility.GetOuterWidth(reference);
```

Comment:
Because we are changing the related code it would be good if we extract `LayoutUtility.GetOuterWidth(reference)` into a variable so that we do not call it three times in a row. The method is comparatively slow since it changes CSS properties of the element to calculate the width.